### DISCUSSION TITLE
Great Project!

### DISCUSSION THREAD

**Discussion by 0xroylee** [UM-ARK/UM-All-Frontend#154] [General]
Hi, guys

I am Roy. I am a full stack developer and graduated 2 years ago from UM. I'd like to have a conversation with your guys about anything I can help or support.

My WeChat ID is: Rweb3y.

I am curious to know why don't use Expo and better not to use the feature-based structure for a small project.

Best
Roy

**Comment by yyyyyyounger** [comment]
Trying to explore expo features, learning...😤
